dotnet/src/Connectors/Connectors.OpenAI/Files/OpenAIFilePurpose.cs

```diff
@@ -8,15 +9,90 @@ namespace Microsoft.SemanticKernel.Connectors.OpenAI;
 /// Defines the purpose associated with the uploaded file.
 /// </summary>
 [Experimental("SKEXP0010")]
-public enum OpenAIFilePurpose
+public readonly struct OpenAIFilePurpose : IEquatable<OpenAIFilePurpose>
```

Comment:
It would be great to add links to XML documentation with API reference of possible values on OpenAI website.

Reply:
Updated